pkg/ddevapp/ddevapp.go

```diff
@@ -127,6 +127,7 @@ type DdevApp struct {
 	DisableSettingsManagement bool                   `yaml:"disable_settings_management,omitempty"`
 	WebEnvironment            []string               `yaml:"web_environment"`
 	NodeJSVersion             string                 `yaml:"nodejs_version,omitempty"`
+	CorepackEnable            bool                   `yaml:"corepack_enable"`
```

Comment:
```suggestion
	CorepackEnable            bool                   `yaml:"corepack_enable,omitempty"`
```

Reply:
I prefer to have this in the config I think, rather than disappear. I usually prefer to `omitempty` things that almost nobody ever needs to know about, but this one seems like it will want to be visible.

Reply:
OK, thanks.